src/main/frontend/components/page.css

```diff
@@ -363,11 +363,6 @@ html.is-native-ios {
   }
  }
 
- .text-2xs {
-  font-size: 0.625rem; /* 10px */
-  line-height: 0.875rem; /* 14px */
- }
-
```

Comment:
Replaced by [this](https://github.com/logseq/logseq/pull/6821/files#r978539637).